### PR TITLE
[APP-534] $ bug fix

### PR DESCRIPTION
**Pull request by solaleye**
Background: Subscripts are indicated by a $ on either side of the subscript text.

Current: See Paragraph 4. Subscript shows "R." Then click Edit and look at Paragraph 4. You will see that subscript is actually "IRB" ($IRB$) not "R"

Request: Change Subscript to Show "IRB" and ensure that whatever text is between the $ $ is what displays as subscript.

### CODE REVIEW

**Comment by koriroys** [comment]
@solaleye It's difficult to tell what change you made. Can you provide a line number to narrow in on?

File: lib/marked.js [hunk 1]
```diff
@@ -4,1307 +4,1306 @@
...
+    link: /^!?\[(inside)\]\(href\)/,
+    reflink: /^!?\[(inside)\]\s*\[([^\]]*)\]/,
+    nolink: /^!?\[((?:\[[^\]]*\]|[^\[\]])*)\]/,
+    strong: /^__([\s\S]+?)__(?!_)|^\*\*([\s\S]+?)\*\*(?!\*)/,
+    em: /^\b_((?:[^_]|__)+?)_\b|^\*((?:\*\*|[\s\S])+?)\*(?!\*)/,
+    code: /^(`+)\s*([\s\S]*?[^`])\s*\1(?!`)/,
+    br: /^ {2,}\n(?!\s*$)/,
+    del: noop,
+    sup: /^(\^+)\s*([\s\S]*?[^\^])\s*\1(?!\^)/,
+    sub: /^(\$+)(\S[^\$,]*)\1/,
```
Comment (solaleye) [inline lib/marked.js]: @koriroys , here is the line as requested.

Comment (koriroys) [reply lib/marked.js]:
```ruby
# old
sub: /^(\$+)\S([^\$]*)\S\1/,
# new
sub: /^(\$+)(\S[^\$,]*)\1/,
```

Comment (koriroys) [reply lib/marked.js]: old
screen shot 2019-03-05 at 10 45 57 am

new
screen shot 2019-03-05 at 10 46 33 am

Comment (solaleye) [reply lib/marked.js]: Correct @koriroys!

Comment (solaleye) [reply lib/marked.js]: @koriroys , please test with this `with less than $2,000,000 in Adjusted Net Capital, $6,000 for each remote location` and you will see the difference. Thanks.

Comment (koriroys) [reply lib/marked.js]: @solaleye neither matches

Comment (solaleye) [reply lib/marked.js]: That's the expectation. Thanks.

**Pull request merged**